tests.h

```diff
@@ -26,7 +28,7 @@ struct TestResult {
   t(&failed); \
   if (failed == 0) { \
     tr->passed++; \
-    printf("PASS " #t "\n"); \
+    if(runverbose) printf("PASS " #t "\n"); else printf("."); \
```

Comment:
print out `......` if we're in quiet mode just so we can see the tests running.  This is how I remember jUnit working so could be historical bias :-)